numpy/random/src/pcg64/pcg64.orig.h

```diff
@@ -764,7 +764,7 @@ inline void pcg_setseq_128_srandom_r(struct pcg_state_setseq_128 *rng,
  *
  *     (Note that using modulo is only wise for good RNGs, poorer RNGs
  *     such as raw LCGs do better using a technique based on division.)
- *     Empricical tests show that division is preferable to modulus for
+ *     Empirical tests show that division is preferable to modulus for
```

Comment:
This file is vendored and should be left alone to match the upstream version.

Reply:
pcg-c/include/pcg_variants.h